src/ServicePulse.Host/vue/src/components/failedmessages/MessageView.vue

```diff
@@ -87,7 +87,7 @@ async function archiveMessage() {
   changeRefreshInterval(1000); // We've started an archive, so increase the polling frequency
   try {
     const response = await useArchiveMessage([id.value]);
-    if (response !== undefined) {
+    if (response.data.ok) {
```

Comment:
This _must_ be `ok` or we would have thrown, right?

Reply:
You are right @mikeminutillo.
I refactored the code to get rid of that, and also removed the catch->console.log while I was there.